src/objectid.ts

```diff
@@ -292,9 +293,10 @@ export class ObjectId extends BSONValue {
    * @param time - an integer number representing a number of seconds.
    */
   static createFromTime(time: number): ObjectId {
-    const buffer = ByteUtils.fromNumberArray([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
+    const buffer = ByteUtils.allocate(12);
+    for (let i = 11; i >= 4; i--) buffer[i] = 0;
     // Encode time into first 4 bytes
-    BSONDataView.fromUint8Array(buffer).setUint32(0, time, false);
+    NumberUtils.setInt32BE(buffer, 0, time);
```

Comment:
Is this supposed to be `NumberUtils.setUInt32BE`?

Reply:
https://github.com/mongodb/js-bson/blob/NODE-5934-readDoubleLE/src/utils/number_utils.ts#L65

There is no difference in how you set signed and unsigned, the bytes are the same, it is how they are interpreted when they are read that changes.